examples/preprocessing/plot_discretization_classification.py

```diff
@@ -61,33 +61,52 @@ def get_name(estimator):
 
 
 # list of (estimator, param_grid), where param_grid is used in GridSearchCV
+# The parameter spaces in this example are limited to a narrow band to speed up.
```

Comment:
You can introduce more line breaks instead of shortening the comment :) I think it kinda needs a `its runtime` at the end of the sentence to make it read nicely.

Reply:
Done. I was thinking this comment could be moved into the docstring at the start of the file. What do you think?

Reply:
No I think it's good here, right before where it happens.